cmd/Dockerfile

```diff
@@ -1,6 +1,6 @@
 # Multi-stage dockerfile building a container image with both binaries included
 
-FROM golang:1.16 as builder
+FROM docker.io/library/golang:1.16 as builder
```

Comment:
why docker ? isn't there a quay img ? 

Is there an advantage to using docker registry I'm unaware of ? 

If there a disadvantage of using quay registry I'm unaware of ?

Reply:
Quay is better to use, but this PR only change from short name to full name in order to support podman (basically using the same image but support both podman and docker).
Follow PRs can move to quay.
There is https://quay.io/repository/projectquay/golang?tab=tags but I don't know if it is the same, and currently focused on adapting to podman.

Reply:
Would you mind changing this in one go ?

Reply:
pushed, lets see if it works